kolibri/logger/test/factory_logger.py

```diff
@@ -12,8 +11,6 @@ class Meta:
         model = models.ContentSessionLog
 
     user = factory.SubFactory(FacilityUserFactory)
-    content_id = uuid.uuid4().hex
-    channel_id = uuid.uuid4().hex
     start_timestamp = datetime.datetime.now()
```

Comment:
Seems factory boy was using the same `content_id` and `channel_id` upon creation so the unique `id` constraint was broken in morango. Had to explicitly define define different uuids upon creation of models in the tests.